website/docs/r/asg.html.markdown

```diff
@@ -49,9 +49,9 @@ The following arguments are supported:
   - `destination` - (Required, String) The IP address or CIDR block that can receive traffic.
   - `ports` - (Required, String) A single port, comma-separated ports or range of ports that can receive traffic.
```

Comment:
@psycofdj Please consider removing as discussed in https://github.com/mevansam/terraform-provider-cf/pull/76/files/5035d55096abebfa482ecfe19218ff23b2814ce4#r192446675